pkg/kubectl/cmd/util/helpers.go

```diff
@@ -455,3 +456,39 @@ func UpdateObject(info *resource.Info, updateFn func(runtime.Object) error) (run
 
 	return info.Object, nil
 }
+
+func GetThirdPartyGroupVersions(c *client.RESTClient) ([]unversioned.GroupVersion, []unversioned.GroupVersionKind, error) {
```

Comment:
Godoc please.  In my first scan of this function it looks extremely similar to the `DiscoveryClient`